src/components/molecules/EarthEditor/OutlinePane/index.tsx

```diff
@@ -84,6 +85,7 @@ const OutlinePane: React.FC<Props> = ({
   onLayerImport,
   onLayerGroupCreate,
   onDrop,
+  onZoomToLayer, // TODO
```

Comment:
I'm guessing we don't need the TODO any more :P 
```suggestion
  onZoomToLayer,
```